rpc-darpc/src/main/java/com/ibm/crail/namenode/rpc/darpc/DaRPCNameNodeServer.java

```diff
@@ -2,46 +2,56 @@
 
 import java.net.InetSocketAddress;
 import java.net.URI;
-
 import org.slf4j.Logger;
-
 import com.ibm.crail.conf.CrailConfiguration;
 import com.ibm.crail.rpc.RpcNameNodeService;
 import com.ibm.crail.rpc.RpcServer;
 import com.ibm.crail.utils.CrailUtils;
+import com.ibm.darpc.DaRPCMemPool;
+import com.ibm.darpc.DaRPCMemPoolImpl;
 import com.ibm.darpc.DaRPCServerEndpoint;
 import com.ibm.darpc.DaRPCServerGroup;
 import com.ibm.disni.rdma.RdmaServerEndpoint;
 
 public class DaRPCNameNodeServer extends RpcServer {
 	private static final Logger LOG = CrailUtils.getLogger();
-	
+
 	private RpcNameNodeService service;
 	private DaRPCServerGroup<DaRPCNameNodeRequest, DaRPCNameNodeResponse> namenodeServerGroup;
-	private RdmaServerEndpoint<DaRPCServerEndpoint<DaRPCNameNodeRequest, DaRPCNameNodeResponse>> namenodeServerEp;	
-	
+	private RdmaServerEndpoint<DaRPCServerEndpoint<DaRPCNameNodeRequest, DaRPCNameNodeResponse>> namenodeServerEp;
+
 	public DaRPCNameNodeServer(RpcNameNodeService service){
 		this.service = service;
 		this.namenodeServerEp = null;
 		this.namenodeServerGroup = null;
-	}	
+	}
 
 	public void init(CrailConfiguration conf, String[] args) throws Exception{
 		DaRPCConstants.updateConstants(conf);
 		DaRPCConstants.verify();
-		
+
+		DaRPCMemPool<DaRPCServerEndpoint<DaRPCNameNodeRequest, DaRPCNameNodeResponse>, DaRPCNameNodeRequest, DaRPCNameNodeResponse> memPool = new DaRPCMemPoolImpl<DaRPCServerEndpoint<DaRPCNameNodeRequest, DaRPCNameNodeResponse>, DaRPCNameNodeRequest, DaRPCNameNodeResponse>(
+				DaRPCConstants.NAMENODE_DARPC_MEMPOOL_HUGEPAGEPATH,
+				DaRPCConstants.NAMENODE_DARPC_MEMPOOL_ALLOCSZ,
+				DaRPCConstants.NAMENODE_DARPC_MEMPOOL_ALIGNMENT,
+				DaRPCConstants.NAMENODE_DARPC_MEMPOOL_ALLOC_LIMIT
+				);
 		String _clusterAffinities[] = DaRPCConstants.NAMENODE_DARPC_AFFINITY.split(",");
```

Comment:
I would prefer doing all parsing in the DaRPCConstants.

Reply:
I did not add any parsing here. If you mean the split(), this is old code. I would prefercleaning aup existing code in a separate PR.

Reply:
Sounds good.